OpenRA.Mods.Common/Widgets/ColorMixerWidget.cs

```diff
@@ -32,6 +32,7 @@ public class ColorMixerWidget : Widget
 
 		bool update;
 		object syncWorker = new object();
+		object frontWorker = new object();
```

Comment:
Maybe throw some ```readonly``` modifiers on this and ```syncWorker``` too since dedicated lock objects never change.